extensions/amp-story/1.0/amp-story-access.js

```diff
@@ -1,3 +1,4 @@
+import * as Preact from '#core/dom/jsx';
```

Comment:
Nit: I'm a bit worried we'll confuse this with actual Preact (and the ability to diff). Can we use a different name and `@jsxPragma`?

```js
// input
/** @jsx StaticTree.createElement */

<div/>



// output
StaticTree.createElement("div");
```

[Babel](https://babeljs.io/repl/#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&corejs=3.6&spec=false&loose=false&code_lz=JYWwDg9gTgLgBAKjgQwM5wMo2TYBjAFSgFNi4AzKCEOAcgGI9piB6AE2pYCtUAPWgNwAoISwRIAAj16ZsuQiWIA6PCRzEAogBtiIYgDt4CFkIA8bYADcWAPiFA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=react&prettier=false&targets=&version=7.15.8&externalPlugins=&assumptions=%7B%7D)

Reply:
Using `JsxStaticTree`, with the rule enforcing `Jsx` if not `Preact` for the sake of simplicity. (Without the prefix we'd need to parse the pragma annotation, which would be fragile.)

Reply:
I really wanted to do this but Closure is not playing nice. 

```
WARNING - [JSC_BAD_JSDOC_ANNOTATION] Parse error. illegal use of unknown JSDoc tag "jsx"; ignoring it.
Place another character before the @ to stop JSCompiler from parsing it as an annotation.
```

Tracking: #36680